Makefile

```diff
@@ -61,11 +60,10 @@ clean:
 	rm -f $(MICROSERVICES)
 
 test:
-	go test -cover ./...
-	go vet ./...
+	GO111MODULE=on go test -cover ./...
```

Comment:
Why is this not just `$(GO)`?

Reply:
I get zmq test error when I do that. Probably something to do with the CGO enabled=off. Note we build the services relying on zeromq with CGO enabled.

Reply:
Understood, that makes sense